src/core/reducers/addons.js

```diff
@@ -172,6 +172,8 @@ export function createInternalAddon(
         updateURL: apiAddon.theme_data.updateURL,
         version: apiAddon.theme_data.version,
       }),
+      // We should be using this property.
```

Comment:
I think it would be better to put a comment up above where `theme_data` gets merged into `addon` saying "use addon.themeData[themeProp] instead of addon[themeProp]"